### ISSUE TITLE
Error when doing GraphQL 

### ISSUE THREAD

**Issue by aodihis** [verbb/vizy#161]
**Description**
I have updated the vizy to version 2.0.3 and I got this kind of error when do graphQL. Look like the Craft is not imported on that `VizyImageNodeInterface` class.

```
2022-07-04 08:25:46 [web.ERROR] [Error] Error: Class "verbb\vizy\gql\interfaces\Craft" not found in /var/www/html/vendor/verbb/vizy/src/gql/interfaces/VizyImageNodeInterface.php:21
Stack trace:
#0 /var/www/html/vendor/verbb/vizy/src/gql/types/generators/VizyNodeGenerator.php(28): verbb\vizy\gql\interfaces\VizyImageNodeInterface::getFieldDefinitions()
#1 /var/www/html/vendor/verbb/vizy/src/gql/interfaces/VizyNodeInterface.php(39): verbb\vizy\gql\types\generators\VizyNodeGenerator::generateTypes()
#2 /var/www/html/vendor/verbb/vizy/src/gql/types/NodeCollectionType.php(53): verbb\vizy\gql\interfaces\VizyNodeInterface::getType()
#3 /var/www/html/vendor/verbb/vizy/src/fields/VizyField.php(503): verbb\vizy\gql\types\NodeCollectionType::getType()
#4 /var/www/html/vendor/craftcms/commerce/src/gql/types/generators/ProductType.php(50): verbb\vizy\fields\VizyField->getContentGqlType()
#5 /var/www/html/vendor/craftcms/commerce/src/gql/interfaces/elements/Product.php(52): craft\commerce\gql\types\generators\ProductType::generateTypes()
#6 /var/www/html/vendor/craftcms/commerce/src/fields/Products.php(69): craft\commerce\gql\interfaces\elements\Product::getType()
#7 /var/www/html/vendor/craftcms/cms/src/gql/base/Generator.php(47): craft\commerce\fields\Products->getContentGqlType()
#8 /var/www/html/vendor/verbb/super-table/src/gql/types/generators/SuperTableBlockType.php(50): craft\gql\base\Generator::getContentFields()
#9 /var/www/html/vendor/verbb/super-table/src/gql/types/generators/SuperTableBlockType.php(34): verbb\supertable\gql\types\generators\SuperTableBlockType::generateType()
#10 /var/www/html/vendor/verbb/super-table/src/gql/interfaces/elements/SuperTableBlock.php(39): verbb\supertable\gql\types\generators\SuperTableBlockType::generateTypes()
#11 /var/www/html/vendor/verbb/super-table/src/gql/types/elements/SuperTableBlock.php(19): verbb\supertable\gql\interfaces\elements\SuperTableBlock::getType()
#12 /var/www/html/vendor/verbb/super-table/src/gql/types/generators/SuperTableBlockType.php(61): verbb\supertable\gql\types\elements\SuperTableBlock->__construct()
#13 /var/www/html/vendor/verbb/super-table/src/gql/types/generators/SuperTableBlockType.php(34): verbb\supertable\gql\types\generators\SuperTableBlockType::generateType()
#14 /var/www/html/vendor/verbb/super-table/src/fields/SuperTableField.php(960): verbb\supertable\gql\types\generators\SuperTableBlockType::generateTypes()
#15 /var/www/html/vendor/craftcms/cms/src/gql/base/Generator.php(47): verbb\supertable\fields\SuperTableField->getContentGqlType()
#16 /var/www/html/vendor/craftcms/cms/src/gql/types/generators/EntryType.php(58): craft\gql\base\Generator::getContentFields()
#17 /var/www/html/vendor/craftcms/cms/src/gql/types/generators/EntryType.php(40): craft\gql\types\generators\EntryType::generateType()
#18 /var/www/html/vendor/craftcms/cms/src/gql/interfaces/elements/Entry.php(54): craft\gql\types\generators\EntryType::generateTypes()
#19 /var/www/html/vendor/craftcms/cms/src/gql/queries/Entry.php(40): craft\gql\interfaces\elements\Entry::getType()
#20 /var/www/html/vendor/craftcms/cms/src/services/Gql.php(1324): craft\gql\queries\Entry::getQueries()
#21 /var/www/html/vendor/craftcms/cms/src/services/Gql.php(366): craft\services\Gql->_registerGqlQueries()
#22 /var/www/html/vendor/craftcms/cms/src/services/Gql.php(501): craft\services\Gql->getSchemaDef()
#23 /var/www/html/vendor/craftcms/cms/src/controllers/GraphqlController.php(177): craft\services\Gql->executeQuery()
#24 [internal function]: craft\controllers\GraphqlController->actionApi()
#25 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array()
#26 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams()
#27 
```

**Steps to reproduce**
1.  Update to version 2.0.3
2. Do any command GraphQL

**Additional info**
- Plugin version:2.0.3
- Craft version:4.1.0.2
- Multi-site: Yes or No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-4 as 2.0.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.4](https://github.com/verbb/vizy/releases/tag/2.0.4)
